lib/Domain/Helper/TemplatePathsResolver.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Phpactor\CodeBuilder\Domain\Helper;
+
+use FilesystemIterator;
+
+class TemplatePathsResolver
+{
+    /**
+     * @var string In the form of "major.minor.release[extra]"
```

Comment:
Is this a valid docblock? Otherwise I'd be happy to ommit the `@var`

Reply:
I think so: https://docs.phpdoc.org/references/phpdoc/tags/var.html

Reply:
> The @var tag MUST contain the name of the element it documents. An exception to this is when property declarations only refer to a single property. In this case the name of the property MAY be omitted.

Only can be omitted for properties at the moment.